mitosheet/mitosheet/public/v3/sheet_functions/README.md

```diff
@@ -0,0 +1,28 @@
+# Guide to Writing Sheet Functions
+
+## Context on Formulas
+From frontend to backend (in order), a formula will have the following order of operations:
+1. The user edits a cell to include a formula. They'll see some information in a dropdown below the cell editor that's autogenerated from a docstring in the python function for that formula. If you edit that docstring or create a new function, you'll want to run `mitosheet/docs/make_function_docs.py update_frontend` to update the information displayed in that dropdown. 
+2. Validation is run on the frontend to check for things like the validity of the columns used as input and whether the formula exists.
```

Comment:
This happens on the backend, in the parser!

Reply:
Ah oops